### ISSUE TITLE
Compute embeddings for various vehicle makes/models?

### ISSUE THREAD

**Issue by BadWindshield** [omoindrot/tensorflow-triplet-loss#32]
Hi. I am new to this repo. I was able to get the MNIST demo running quickly. Nice work!

I am interested in training my own embeddings for various vehicle makes/models. For example, there is the "Cars" dataset [ http://ai.stanford.edu/~jkrause/cars/car_dataset.html ] where annotations are available for the model/make of a vehicle.

My question is -- is the CNN architecture here general enough to handle vehicles? Do I need something more sophisticated like the VGG 1024 CNN in Faster RCNN? Thanks very much.

**Comment by omoindrot** [comment]
The current model in this repo is very small so it works for MNIST but it won't work for bigger images.

I would recommend reshaping your images to `224*224` and use a pre-trained network as a starting point like resnet.

**Comment by BadWindshield** [comment]
@omoindrot Thanks!
